modules/exploits/linux/local/su_login.rb

```diff
@@ -0,0 +1,97 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Local
+    include Msf::Post::Linux
+    include Msf::Post::Linux::System
+    include Msf::Post::Unix
+    include Msf::Post::File
+    include Msf::Exploit::FileDropper
+
+    Rank = NormalRanking
+
+    def initialize(info = {})
+        super(
+            update_info(info,
+                "Name"           => "Login to Another User with Su on Linux / Unix Systems",
+                "Description"    => "This module attempts to create a new login session by
+                                     invoking the su command of a valid username and password.
+
+                                     If the login is successful, a new session is created via
+                                     the specified payload.",
+                "License"        => MSF_LICENSE,
+                "Author"         => "Gavin Youker <youkergav@gmail.com>",
+                "DisclosureDate" => "Sep 10 2020",
```

Comment:
The disclosure date is usually the date the technique was discovered; or, in instances such as this where the functionality is intended functionality, the date should be the date the functionality was developed, which is:

```suggestion
                "DisclosureDate" => "1971-11-03",
```

Reply:
Fixed. See commit 38fb6441696662d9aea17d95c8bc30021e002088